examples/tutorials/lighting/app.ts

```diff
@@ -89,6 +90,10 @@ void main(void) {
   vPosition = (app.modelMatrix * vec4(positions, 1.0)).xyz;
   vNormal = mat3(app.modelMatrix) * normals;
   vUV = texCoords;
+
+  #if (defined(LIGHTING_VERTEX))
```

Comment:
Nit: Can we use simpler syntax `#ifdef` instead of `#if defined()`? Can help to to make WGSL and GLSL code more equivalent.